tests/e2e/features/createSpace.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Create a space
+    As a super user
+    I want to create a space
+    So that I can add recipes and share my recipes
+
+    Background:
+        Given the user has signed up with the following details
+            | username         | superuser |
+            | password         | super1234 |
```

Comment:
I'm a little bit confused here is this "superuser" just a normal user named "superuser" or is there a concept of "admin" ie. superuser in this system?

Reply:
> I'm a little bit confused here is this "superuser" just a normal user named "superuser" or is there a concept of "admin" ie. superuser in this system?

yes superuser is the admin. we have written tests only for the admin for now